src/io/aviso/taxi_toolkit/assertions.clj

```diff
@@ -36,9 +36,8 @@
 
 (defn has-class?
   "Indicates whether element has a given class applied"
-  [css-class]
-  (fn [el]
-    (some #{css-class} (classes el))))
+  [css-class el]
```

Comment:
I believe has-class? should return either true/false instead of a function that does that. I don't feel strongly about it though, it felt intuitive and that's the way I'd like to use it in suite.